### PR TITLE
[TACHYON-1397] Add examples to Tachyon Shell documentation

### PR DESCRIPTION
**Pull request by calvinjia**
https://tachyon.atlassian.net/browse/TACHYON-1397

### CODE REVIEW

File: docs/Command-Line-Interface.md [hunk 1]
```diff
@@ -20,6 +20,32 @@ Or, if no header is provided, the default hostname and port (set in the env file
 
     /<path>
 
+## Wildcard Input
+
+Most of the commands which require path components allow wildcard arguments for ease of use. For
+example:
+
+```
```
Comment (apc999) [inline docs/Command-Line-Interface.md]: NIT: To keep consistent with other docs:
could you add syntax hint?
<pre>```bash</pre> 
also put a prompt `$` in beginning of each command line?

File: docs/Command-Line-Interface.md [hunk 2]
```diff
@@ -182,3 +209,335 @@ Or, if no header is provided, the default hostname and port (set in the env file
     <td>Remove the TTL (time to live) setting from a file.</td>
   </tr>
 </table>
+
+# Example Use Cases
+
+## cat
+The `cat` command will print the entire contents of a file in Tachyon to the console. This can be
```
Comment (apc999) [inline docs/Command-Line-Interface.md]: NIT: most other examples uses present tense as the first sentence (e.g., fileinfo and below), while `cat`,  `copyFromLocal` and a few uses future tense. Shall we keep consistency across this section?

File: docs/Command-Line-Interface.md [hunk 3]
```diff
@@ -20,6 +20,32 @@ Or, if no header is provided, the default hostname and port (set in the env file
...
+```
+bin/tachyon tfs rm /data/2014*
+```
+
+The example command would delete anything in the `data` directory with a prefix of `2014`.
+
+Note that some shells will attempt to glob the input paths, causing strange errors like
+
+```
+rm takes 1 arguments,  not 21
```
Comment (apc999) [inline docs/Command-Line-Interface.md]: I was confused by the number 21 and actually spent some time thinking why this number comes here. Could you explain a little bit (e.g., there are 21 other files in local file systems matched?)

**Comment by apc999** [comment]
Thanks for improving the doc! I left a few minor suggestions.

File: docs/Command-Line-Interface.md (hunk 1 above)
Comment (calvinjia) [inline docs/Command-Line-Interface.md]: Updated, thanks

File: docs/Command-Line-Interface.md (hunk 2 above)
Comment (calvinjia) [inline docs/Command-Line-Interface.md]: Good idea, I've changed it all to present tense.

File: docs/Command-Line-Interface.md (hunk 3 above)
Comment (calvinjia) [inline docs/Command-Line-Interface.md]: Sure, updated.

**Comment by calvinjia** [comment]
@apc999 I updated the docs, could you take another look please?

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**